### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by priyankarpal**
Added new styles
fixed wrong spellings

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5206"></a>

### CODE REVIEW

**Review by SurajPratap10 — APPROVED** [review]
lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,5 +43,4 @@ We welcome everyone to review Pull Requests, it is a great way to learn and netw
 - do not repeat feedback, this creates more noise than value (check the existing conversation)
 
 ---
-
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:46]: This blank line is recommended to be kept

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Before working on an existing Issue please follow these steps:
 1. do **not** enable GitHub Actions on your fork
 1. reference the Issue in your Pull Request (for example `closes #123`)
 
-Notes:
+> Notes:
 - any issues that have no activity after 2 weeks will be unassigned and re-assigned to someone else
 - only request to be assigned an issue if you know how to work on it
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:30]: Probably best to have the whole block in the same formatting

```suggestion
> Notes:
> - any issues that have no activity after 2 weeks will be unassigned and re-assigned to someone else
> - only request to be assigned an issue if you know how to work on it
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you 👍 

I added some inline comments

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,4 +43,6 @@ We welcome everyone to review Pull Requests, it is a great way to learn and netw
 - do not repeat feedback, this creates more noise than value (check the existing conversation)
 
 ---
+
+
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:47]: Single blank line is recommended between Markdown sections

```suggestion

```

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**